Tutorials/MODPROD21/Tutorial2/Solutions/QuarterCarModel/temp-excercise1/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
+*
+# Except this file
+!.gitignore
```

Comment:
Why pusing an empty directory?

Reply:
Otherwise the user has to create it first before starting the notebooks. I think OMPython throws an error when using `cd("temp-exercise2")`.